CMakeLists.txt

```diff
@@ -236,3 +236,56 @@ if(SDFLIB_BUILD_DEBUG_APPS)
     # target_link_libraries(ImageQueryTime PUBLIC ${PROJECT_NAME} CGAL::CGAL OpenVDB::openvdb)
     target_link_libraries(ImageQueryTime PUBLIC ${PROJECT_NAME})
 endif()
+
+if(SDFLIB_USE_SYSTEM_GLM)
+    list(APPEND SDFLIB_DEPENDENCIES glm)
+endif()
+
+if(SDFLIB_USE_SYSTEM_SPDLOG)
+    list(APPEND SDFLIB_DEPENDENCIES spdlog)
+endif()
+
+if(SDFLIB_USE_SYSTEM_CEREAL)
+    list(APPEND SDFLIB_DEPENDENCIES cereal)
+endif()
+
+if(SDFLIB_USE_SYSTEM_ASSIMP)
+    list(APPEND SDFLIB_DEPENDENCIES assimp)
+endif()
+
+if(SDFLIB_USE_OPENMP)
+    list(APPEND SDFLIB_DEPENDENCIES OpenMP Threads)
```

Comment:
Those package names won't work with `find_dependency(<name> CONFIG)`.

Reply:
I will change it to find package with condition